### ISSUE TITLE
CT xAlign:stretch can only be used if WidthSizable is mixed into the content

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/sun#780]
```
sun : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660563931656%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660569734366%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : multitouch-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660564193622%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=false
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : pan-and-zoom-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660571823437%7D&brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=true
Query: brand=phet&ea&fuzz&fuzzPointers=2&memoryLimit=1000&supportsPanAndZoom=true
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-api-compatibility : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-api-compatibility%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660563305806%7D&ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Query: ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-api-compatibility : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-api-compatibility%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660570815118%7D&ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Query: ea&brand=phet-io&phetioStandalone&phetioCompareAPI&randomSeed=332211
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660563396878%7D&ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Query: ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sun/sun_en.html?continuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660570209136%7D&ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Query: ea&brand=phet-io&phetioStandalone&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/phet-io-wrappers/state/?sim=sun&phetioDebug=true&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660563329736%7D
Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-state-fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/phet-io-wrappers/state/?sim=sun&phetioDebug=true&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22sun%22%2C%22phet-io-state-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1660558781438%22%2C%22timestamp%22%3A1660570163704%7D
Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-wrappers-tests : assert
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/phet-io-wrappers/phet-io-wrappers-tests.html?sim=sun&phetioDebug=true&phetioWrapperDebug=true
32 out of 33 tests passed. 1 failed.
SimTests: sun: event indexing failed:
Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at AlignBoxConstraint.layout (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:634:19)
at AlignBoxConstraint.updateLayout (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/constraints/LayoutConstraint.js:185:14)
at AlignBox.invalidateAlignment (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:135:21)
at AlignBox.setXAlign (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:225:12)
at AlignBox.set xAlign [as xAlign] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:232:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/nodes/Node.js:6370:21
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at AlignBox.mutate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/nodes/Node.js:6357:7)

Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the contentError: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM

----------------------------------

sun : phet-io-wrappers-tests : no-assert
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/phet-io-wrappers/phet-io-wrappers-tests.html?sim=sun
30 out of 31 tests passed. 1 failed.
SimTests: sun: event indexing failed:
Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at AlignBoxConstraint.layout (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:634:19)
at AlignBoxConstraint.updateLayout (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/constraints/LayoutConstraint.js:185:14)
at AlignBox.invalidateAlignment (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:135:21)
at AlignBox.setXAlign (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:225:12)
at AlignBox.set xAlign [as xAlign] (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/layout/nodes/AlignBox.js:232:10)
at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/nodes/Node.js:6370:21
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at AlignBox.mutate (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/chipper/dist/js/scenery/js/nodes/Node.js:6357:7)

Uncaught Error: Uncaught Error: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the contentError: Assertion failed: xAlign:stretch can only be used if WidthSizable is mixed into the content
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/assert/js/assert.js:28:13)
at assert (AlignBox.ts:690:18)
at layout (LayoutConstraint.ts:189:13)
at updateLayout (AlignBox.ts:193:20)
at invalidateAlignment (AlignBox.ts:280:11)
at setXAlign (AlignBox.ts:286:52)
at (Node.ts:6342:10)
at arrayEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:537:11)
at Function.forEach (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1660558781438/sherpa/lib/lodash-4.17.4.js:9359:14)
at each (Node.ts:6325:6)
id: Bayes Puppeteer
Snapshot from 8/15/2022, 4:19:41 AM
```

**Comment by jbphet** [comment]
This is easy to duplicate using the sun demo.  Here are the steps:

1. Load the demo
2. Select the Layout screen
3. Select "Sizable buttons" from the combo box

I spent a little time investigating this and, while I understand the gist of the error message, I don't understand what has changed that is causing this error to start occurring, nor do I know what should be done about it.  Since this seems very related to the recent changes to `LayoutNode`, I pinged @jonathanolson (the author of those change) via Slack in order to ask him about it, but it seems like he is not around.  My guess is that he'll recognize what is up right away, so I'm assigning this issue to him.
